src/components/Book.js

```diff
@@ -21,22 +29,16 @@ const Book = (props) => {
           <p className="para" style={styles.para}>{book.title}</p>
           <p>{book.author}</p>
           <span>
-            <button type="button">Remove</button>
+            {/* <button type="button">Comments</button> */}
+            <button type="button" onClick={() => handleRemove(book.id)}>
+              Remove
+            </button>
+            {/* <button type="button">Edit</button> */}
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧